srcpkgs/tap-plugins/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'tap-plugins'
+pkgname=tap-plugins
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+
+pre_build() {
+	make_build_args="CC=${CC} LD=$(LD)"
```

Comment:
`LD=$(LD)` Will try to execute LD, you probably want `LD=${LD}`.

Reply:
My bad, missed this typo! Fixed